conduce/api.py

```diff
@@ -748,8 +748,10 @@ def get_transactions(dataset_id, **kwargs):
     # HACK: get around bool parameter bug
     if not kwargs.get('count'):
         parameters.pop('count')
+    if kwargs.get('max'):
+        parameters.pop('min')
```

Comment:
Why?

It is perfectly legal and reasonable to request a range of transactions, which is why a user would set both max and min.

Reply:
Oops, that was not supposed to be included.  #160